save-cosv/src/main/kotlin/com/saveourtool/save/cosv/repository/CosvRepositoryInStorage.kt

```diff
@@ -62,13 +70,15 @@ class CosvRepositoryInStorage(
                 if (existedMetadata.user.requiredId() != user.requiredId()) {
                     throw ResponseStatusException(
                         HttpStatus.FORBIDDEN,
```

Comment:
Should it be `FORBIDDEN`?

Reply:
yes, if someone uploaded a vulnerability, only this user can update it (in phase one)